go-selinux/selinux_linux_test.go

```diff
@@ -412,6 +413,76 @@ func TestGlbLub(t *testing.T) {
 	}
 }
 
+func TestContextWithLevel(t *testing.T) {
+	want := "bob:sysadm_r:sysadm_t:SystemLow-SystemHigh"
+
+	verifier := func(con string) error {
+		if con != want {
+			return fmt.Errorf("invalid context %s", con)
+		}
+
+		return nil
+	}
+
+	userBuff := `# COMMENT
+foo_r:foo_t:s0     sysadm_r:sysadm_t:s0
+staff_r:staff_t:s0                 baz_r:baz_t:s0   sysadm_r:sysadm_t:s0
+`
+
+	defaultBuff := `
+foo_r:foo_t:s0     sysadm_r:sysadm_t:s0
+staff_r:staff_t:s0                 baz_r:baz_t:s0   sysadm_r:sysadm_t:s0
+`
+
+	c := defaultSECtx{
+		user:       "bob",
+		level:      "SystemLow-SystemHigh",
+		scon:       "system_u:staff_r:staff_t:s0",
+		userRdr:    bytes.NewBufferString(userBuff),
+		defaultRdr: bytes.NewBufferString(defaultBuff),
```

Comment:
So, these are getting consumed by every test, and I don't see any resets here...